content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -512,33 +501,48 @@ The SNI proxy will forward the traffic to port `443`.
     metadata:
       name: forward-downstream-sni
     spec:
-      filters:
-      - listenerMatch:
-          portNumber: 443
-          listenerType: SIDECAR_OUTBOUND
-        filterName: forward_downstream_sni
-        filterType: NETWORK
-        filterConfig: {}
+      configPatches:
+      - applyTo: NETWORK_FILTER
+        match:
+          context: SIDECAR_OUTBOUND
+          listener:
+            portNumber: 443
+            filterChain:
+              filter:
+                name: mixer
+        patch:
+          operation: INSERT_BEFORE
+          value:
+             name: forward-downstream-sni
+             config: {}
     ---
     # The following filter verifies that the SNI of the mutual TLS connection (the SNI reported to Mixer) is
```

Comment:
Here, the doc uses 'SNI' and elsewhere in this same doc, the 'HOST header' is used. Is that distinction deliberate?

Reply:
@ssuchter  please note that this PR contains changes to two documents:

1. content/en/docs/tasks/traffic-management/egress/egress-control/index.md
1. content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

On the one hand maybe it is not the best practice, on the other hand it provides you more context on the same topic - preventing attacks in which malicious clients cheat Istio by spoofing HOST Header or SNI value. Note that the HOST header and SNI are two different means to specify the destination host, both can be spoofed by malicious clients. Spoofing by the HOST header and by the SNI value is prevented by different means in Istio.

Reply:
Got it. Nevermind!